extra-dev/packages/coq-category-theory/coq-category-theory.dev/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+version: "dev"
+
+homepage: "https://github.com/jwiegley/category-theory"
+dev-repo: "git+https://github.com/jwiegley/category-theory.git"
+bug-reports: "https://github.com/jwiegley/category-theory/issues"
+license: "BSD-3-Clause"
+
+synopsis: "An axiom-free formalization of category theory in Coq"
+description: """
+An axiom-free formalization of category theory in Coq for personal study and
+practical work.
+"""
+
+build: [make "JOBS=%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.10" & < "8.15~") | (= "dev")}
```

Comment:
Since the development is meant to be compiled with Coq's master, just change the specification to plain `"coq" {>= "8.10"}`. There is no reason to exclude 8.15.